packages/cms/src/encrypted_content_info.ts

```diff
@@ -0,0 +1,25 @@
+import { AsnProp, AsnPropTypes, OctetString } from "@peculiar/asn1-schema";
+import { ContentType, ContentEncryptionAlgorithmIdentifier, EncryptedContent } from "./types";
+
+/**
+ * EncryptedContentInfo ::= SEQUENCE {
```

Comment:
Wrap ASN.1 schema comments into MD code block. And do this for all other comments
```diff
/**
+* ```asn
 * EncryptedContentInfo ::= SEQUENCE {
 *  contentType ContentType,
 *  contentEncryptionAlgorithm ContentEncryptionAlgorithmIdentifier,
 *  encryptedContent [0] IMPLICIT EncryptedContent OPTIONAL }
+* ```
 */
 ```